frontend/src/components/Profile/Profile.js

```diff
@@ -1,39 +1,38 @@
-import React, { useState } from "react"
+import React from "react"
 import profilestyle from "./Profile.module.css"
-import GeneratePass from "../GeneratePass/GeneratePass"
-import ViewPasses from "../ViewPasses/ViewPasses"
 import { useNavigate } from "react-router-dom"
 
+//check for role for button display (array element) user, guard, admin
```

Comment:
Remove comments

Reply:
sorry i forgot to remove them, removed now.